### PR TITLE
Add client certificate authentication

### PR DESCRIPTION
**Pull request by grydz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# CHANGELOG
+
+## \[1.1.0\] - 2024-12-19
```
Comment (Manuthor) [inline CHANGELOG.md]:
```suggestion
## [1.1.0] - 2024-12-19
```

File: src/cenclave_lib_sgx/cli.py [hunk 2]
```diff
@@ -241,6 +243,12 @@ def run() -> None:
         "wsgi_max_body_size": 2 * 1024 * 1024 * 1024,  # 2 GB
     }
 
+    if client_cert := args.client_certificate:
+        config_map["verify_mode"] = 2
```
Comment (Manuthor) [inline src/cenclave_lib_sgx/cli.py]: declare a const for this `2` value? (for readability I mean)

**Review by Manuthor — APPROVED** [review]

File: src/cenclave_lib_sgx/cli.py (hunk 2 above)
Comment (grydz) [reply src/cenclave_lib_sgx/cli.py]: Fixed in 55574cb89cdbb4fcb7a7e3cce7cfd59baf7f3f45.

File: CHANGELOG.md (hunk 1 above)
Comment (grydz) [reply CHANGELOG.md]: Fixed in c0d60fa16116790c6e26299acfcaa3b560d71613.

**Pull request merged**